plugins/modules/lke_version_list.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+"""This module allows users to list Linode instances."""
+from __future__ import absolute_import, division, print_function
+
+from typing import Any, Optional, Dict
+
+from ansible_specdoc.objects import SpecDocMeta, SpecReturnValue, FieldType, SpecField
+
+from ansible_collections.linode.cloud.plugins.module_utils.linode_common import LinodeModuleBase
+from ansible_collections.linode.cloud.plugins.module_utils.linode_helper import get_all_paginated
+from ansible_collections.linode.cloud.plugins.module_utils.linode_docs import global_authors, \
+     global_requirements
+
+import ansible_collections.linode.cloud.plugins.module_utils.doc_fragments.lke_version_list as docs
+
+spec = dict(
+    # Disable the default values
+    state=SpecField(type=FieldType.string, required=False, doc_hide=True),
+    label=SpecField(type=FieldType.string, required=False, doc_hide=True),
+
+    order=SpecField(type=FieldType.string, description=['The order to list lke versions in.'],
+               default='asc', choices=['desc', 'asc']),
+    order_by=SpecField(type=FieldType.string,
```

Comment:
`order_by` seems never be used because LKE version is a simple JS obj like `{"id":"1.25"}`, and there isn't another attribute other than the id. Is it possible to remove it?

Reply:
Good catch, I'll take care of it!